package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Hide NSFW content from websites using this extension powered by AI! This extension replaces images that are NSFW with random images from Unsplash.",
   "scripts": {
     "copy": "cp src/content.js dist/src/",
-    "build": "parcel build src/background.js -d dist/src/ -o background && npm run copy",
```

Comment:
Why was this changed?

Reply:
It just build all files in `src` folder, and you dont need to copy files from `src` to `dist/src`, I need cause I do some `import` from `utils.js` and I need to babelify some ES6 code like `import/export`